assets/src/blocks/Accordion/AccordionBlock.js

```diff
@@ -0,0 +1,50 @@
+import { Component } from '@wordpress/element';
+import { AccordionEditor } from './AccordionEditor';
+import { frontendRendered } from '../frontendRendered';
+
+const BLOCK_NAME = 'planet4-blocks/accordion';
+
+export class AccordionBlock extends Component {
+  constructor (props) {
+    super(props);
+    const { registerBlockType } = wp.blocks;
+    const { __ } = wp.i18n;
+
+    const attributes = {
+      title: {
+        type: 'string',
+        default: '',
+      },
+      description: {
+        type: 'string',
+        default: '',
+      },
+      tabs: {
+        type: 'array',
+        default: [],
+      },
+    };
+
+    registerBlockType(BLOCK_NAME, {
+      title: __('Accordion', 'planet4-blocks-backend'),
+      icon: 'menu',
+      category: 'planet4-blocks-beta',
+      keywords: [
+        __(BLOCK_NAME),
+        __('faq'),
+        __('collapsible')
+      ],
+      attributes,
+      edit: AccordionEditor,
+      save: frontendRendered(BLOCK_NAME),
```

Comment:
Do we need to render this on the front end? Can we instead save the html?

Reply:
You mean using directly `AccordionFrontend` here?